packages/salesforcedx-vscode-visualforce-next/typings/vscode.proposed.d.ts

```diff
@@ -0,0 +1,316 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
The idea of embedding this file is from https://github.com/Microsoft/vscode/issues/25784